src/l500/l500-options.cpp

```diff
@@ -18,6 +18,16 @@ namespace librealsense
 
     void l500_hw_options::set(float value)
     {
+        // Block activating alternate IR when IR reflectivity is on [RS5-8358]
+        auto &ds = _l500_dev->get_depth_sensor();
+        if( ( alternate_ir == _type ) && ( 1.0f == value ) )
+        {
+            if( ds.supports_option( RS2_OPTION_ENABLE_IR_REFLECTIVITY )
+                && ds.get_option( RS2_OPTION_ENABLE_IR_REFLECTIVITY ).query() == 1.0f )
+                throw librealsense::wrong_api_call_sequence_exception(
+                    "Enabling Alternate IR while IR Reflectivity is enabled is prohibited" );
```

Comment:
`Alternate IR cannot be enabled with IR Reflectivity`

Reply:
👍